code/src/components/MovieDetails.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react'
+import { useParams, useHistory } from 'react-router-dom'
+
+import './movieDetails.css'
+
+import { DETAILS_URL } from '../utils/urls'
+import { Loading } from '../components/Loading'
+
+
+export const MovieDetails = () => {
+
+	const { movieId } = useParams();
+	const history = useHistory(); 
+
+  	const [details, setDetails] = useState([]);
+	const [loading, setLoading] = useState(true);
+
+	const detailStyling = {
+		backgroundImage: `linear-gradient(rgba(0, 0, 0, 0) -70%, rgb(0, 0, 0) 100%), url(https://image.tmdb.org/t/p/w1280/${details.backdrop_path})`,
+		minHeight: '100vh',
+		backgroundSize: 'cover',
+		backgroundRepeat: 'no-repeat'
+	}
+
+	const goBack = () => {
+		history.push("/")
+	}
+	
+	useEffect(() => {
+		fetch(DETAILS_URL(movieId))
+		.then((res) => res.json())
+		.then((json) => { 
+			setLoading(false)
+			setDetails(json)
+		})
+	},[movieId]);
+
+
+    return (
+	<>
+	{loading && <Loading />} 
+  	<section className="summary-section">
+			<div
+					className="background-container"
```

Comment:
To make it easier to read your code perhaps make the elements more inline.

Reply:
Don't know why it looks like this, but in VScode everything looks fine :)